app/workers/smooch_worker.rb

```diff
@@ -6,11 +6,10 @@ class SmoochWorker
 
   sidekiq_retry_in { |_count, _exception| 20 }
 
-  def perform(json_message, type, app_id, request_type, annotated)
-    annotated = YAML.load(annotated)
+  def perform(message_json, type, app_id, request_type, associated_id = nil, associated_class = nil)
```

Comment:
When we deploy, we may have some jobs in the queue that still follow the old format. We can't let those simply fail. Can you please make sure that the old format is still supported?

Reply:
Good idea, I'll add a temp code for this and we can remove it later